src/utils/keystore.ts

```diff
@@ -27,12 +30,21 @@ const DEFAULTS = {
   }
 }
 
+interface Kdf_params {
+  memlimit_kib?: number,
+  opslimit: number,
+  parallelism: number,
+  salt: string
```

Comment:
```suggestion
  salt?: string
```

let's make this optional in order to not use `Partial<Kdf_Params>`